src/workerd/api/sockets.c++

```diff
@@ -9,8 +9,41 @@
 namespace workerd::api {
 
 
+bool isValidAddress(kj::StringPtr address) {
+  if (address.size() > (255 + 6)) {
+    // RFC1035 states that maximum domain name length is 255 octets. But we also add 6 to support
+    // port numbers in the address.
+    //
+    // IP addresses are always shorter, so we take the max domain length instead.
+    return false;
+  }
+
+  for (int i = 0; i < address.size(); i++) {
+    switch (address[i]) {
+      case '-':
+      case '.':
+      case ':':
+      case '_':
+      case '/': // To enable "unix:/" addresses (only used for testing, we expect oxy to reject this).
```

Comment:
```suggestion
      case '/': // To enable "unix:/" addresses (only used for testing).
```

(keep in mind that oxy is an internal thing. outside contributors won't know what that is)